cpp/test/neighbors/brute_force_c.cu

```diff
@@ -90,48 +302,167 @@ void recall_eval(T* query_data,
                                                min_recall));
 };
 
+template <typename T, typename IdxT, typename bitmap_t = uint32_t>
+void recall_eval_with_filter(T* query_data,
```

Comment:
Can you please make sure these are testing all possible combinations of the filter struct/type? Eg NULL, NO_FILTER, BITMAP, and BITSET? 

We should set a good precedent for testing these early, so that all follow-on filtering implementations can do the same thing.

Reply:
Yeah, that makes sense. I just checked the code, the `NO_FILTER`, `BITMAP` have been covered(pls refer to https://github.com/rapidsai/cuvs/pull/174/files#diff-1b162130023c13d62c06d91f3aab663d943091dddd8a38813c27f65e7d6c61dfR93), and the `BITSET ` is reserved for other algorithms to compatible which is not be implemented in brute-force. Would you like me to implement BITSET in brute-force and test it?